hypervisor/src/hypervisor/syscall.rs

```diff
@@ -147,7 +147,11 @@ mod tests {
 
             // Send the fds to the responder
             {
-                let fds = [requ_fd.get_fd(), resp_fd.get_fd(), event_fd.as_raw_fd()];
+                let fds = [
+                    requ_fd.as_fd().as_raw_fd(),
```

Comment:
Why borrow first?

requ_fs.as_raw_fd() should do the same if implemented?

Reply:
done